openmc/geometry.py

```diff
@@ -41,24 +46,25 @@ def root_universe(self, root_universe):
 
         self._root_universe = root_universe
 
-
     def get_offset(self, path, filter_offset):
-        """
-        Returns the corresponding location in the results array for a given
+        """Returns the corresponding location in the results array for a given
```

Comment:
I think an additional comment should be included that this routine is intended to be used for distribcell tallies - e.g., the path should correspond to a path for a distribcell. I realize this is a deficiency of my original docstring.